src/DustyBot/Modules/BotModule.cs

```diff
@@ -351,6 +354,64 @@ public async Task RemoveSupporter(ICommand command)
             await command.ReplySuccess(Communicator, $"Removed {removed} supporters.");
         }
 
+        [Command("번역", "Translate words and sentences.")]
+        [Alias("tr")]
+        [Parameter("start", ParameterType.String, "Select the language that needs to be translated.")]
+        [Parameter("end", ParameterType.String, "Select the language to translate.")]
+        [Parameter("Message", ParameterType.String, ParameterFlags.Remainder, "Input the word or sentence you want to translate.")]
+        public async Task Translation(ICommand command)
```

Comment:
This doesn't really belong in the `Bot` module. I'm not sure where to put it. Maybe put it into the `Info` module for now? Or create a new `Translator` module.